.moban.cd/moban.yml

```diff
@@ -4,13 +4,17 @@ organisation: moremoban
 author: C. W.
 contact: wangc_2011@hotmail.com
 license: MIT
-version: 0.5.0
-current_version: 0.5.0
+version: 0.6.0
+current_version: 0.6.0
 release: 0.5.0
 branch: master
 master: index
-command_line_interface: "moban"
-entry_point: "moban.main:main"
+entry_points:
+  console_scripts:
+    - "moban = moban.main:main"
+  fs.opener:
+    - "pypi = moban.fs_openers:PypiFSOpener"
+    - "repo = moban.fs_openers:RepoFSOpener"
```

Comment:
these should not be exposed using the fs.opener entry point if they are not designed/implemented to be re-usable outside of moban.  Instead I believe we can install it only within moban using `fs.opener.registry.Registry`

Reply:
I have to check it out. To be honest, I don’t like setup entry point because it impose the test script to install itself to enable the custom fs openers.

Reply:
test scripts do not need to use the setup entry points, and it is not a good idea to do this (it makes packaging more difficult also).  See the docs for `Registry` - it has an automatic registry decorator which can be used.

Reply:
Sure! Will switch to use Registry.